lib/charms/opensearch/v0/opensearch_tls.py

```diff
@@ -549,13 +547,15 @@ def store_new_ca(self, secrets: Dict[str, Any]) -> bool:  # noqa: C901
             ):
                 raise
 
-        with tempfile.NamedTemporaryFile(mode="w+t") as ca_tmp_file:
+        with tempfile.NamedTemporaryFile(
+            mode="w+t", dir=self.charm.opensearch.paths.conf
+        ) as ca_tmp_file:
```

Comment:
:+1: good change. Just for consideration: there is also a `self.charm.opensearch.paths.certs`.

Reply:
Yes, I was between that one or `conf`. I think either way is fine since the files disappear once `close()` is called.